Application/Test.Common/MsSqlDatabaseHelper.cs

```diff
@@ -38,6 +41,49 @@ public void CopyDatabase(string originalDatabaseName, string newDatabaseName)
             }
         }
 
+        public string DownloadAndRestoreDatabase()
+        {
+            try
+            {
+                var psScript = @"
+                    $ErrorActionPreference = 'Stop'
+
+                    .\Initialize-PowershellForDevelopment.ps1
+
+                    $settings = @{ 
+                        ApiSettings = @{ Engine = 'SQLServer' } 
+                        ConnectionStrings = @{ EdFi_Ods = '" + _odsConnectionString + @"' } 
+                    }
+                    Set-DeploymentSettings $settings
+
+                    Reset-TestPopulatedTemplateDatabase
+                ";
+
+                var info = new ProcessStartInfo("powershell", psScript)
+                {
+                    WorkingDirectory = Path.GetFullPath("../../../../../../Ed-Fi-ODS-Implementation"),
```

Comment:
How about if we pass this path as an argument? so this class remains generic, in the future there might be tests with a different folder depth.

Would there be a benefit if we also add this path to `appsettings.json` so that it is configurable?